opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -365,12 +365,34 @@ namespace Opm
 
         const std::vector<int>& resv_wells = SimFIBODetails::resvWells(wells, step, wmap);
 
+        const std::size_t number_resv_wells        = resv_wells.size();
+        std::size_t       global_number_resv_wells = number_resv_wells;
+#if HAVE_MPI
+        if ( solver_.parallelInformation().type() == typeid(ParallelISTLInformation) )
+        {
+            global_number_resv_wells = boost::any_cast<const ParallelISTLInformation&>(solver_.parallelInformation()).communicator().sum(global_number_resv_wells);
```

Comment:
That is a very long line, split it please (perhaps the info object could be in a const ref variable).